### ISSUE TITLE
All calls returning "Unauthorized" - "Access to requested resource is denied."

### ISSUE THREAD

**Issue by nightcoding2021** [amzn/selling-partner-api-models#1433]
Got past signing the request, but all calls to endpoints are then returning the below Unauthorized error.

It appears the API isn't granting access to the locale/selling account pair?  But how on earth to debug?  The locale has an active selling account with SP-API app created.

There is zero information in the response below, so now what?  Is everyone supposed to spend the rest of their days just guessing?

**{
  "errors": [
    {
      "message": "Access to requested resource is denied.",
     "code": "Unauthorized",
     "details": ""
    }
  ]
}**

**Comment by sajal-gupta-76** [comment]
> I Solved This issue by replacing my Grantless Token to LWA Token and it worked for me.

@SohailChohan143 Could you please elaborate on your solution here - "replacing your Grantless Token to LWA Token"

Can you mention the steps involved here.

**Comment by SohailChohan143** [comment]
> > I Solved This issue by replacing my Grantless Token to LWA Token and it worked for me.
> 
> @SohailChohan143 Could you please elaborate on your solution here - "replacing your Grantless Token to LWA Token"
> 
> Can you mention the steps involved here.

Dear @sajal-gupta-76 , I don’t recall the exact question, and I'm unable to see the original message. However, as far as I remember, I was working on Amazon SP-API notifications and subscribing to the Amazon SQS queue for those notifications. Initially, I tried using a Grantless token to subscribe, but it returned a 401 error. I then switched to using an LWA access token, which is typically used to access SP-APIs.
